tcslackbuildnotifier-web-ui/src/test/java/slacknotifications/teamcity/extension/SlackNotificationDMTests.java

```diff
@@ -50,11 +55,153 @@ public void sendToSpecifiedChannel() throws IOException {
+
+    assertNotNull(sendMock(true));
+  }
+
+  @Test
+  public void lessThanFiveOthersTest() throws IOException {
+    controller.addCommitMockNotification(new Commit("jdajfla", "first commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "second commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "early commits", "jdasfj", "fake.slack"));
+    controller.addCommitMockNotification(new Commit("abb23b4", "Merge of branch xyz", "Jimbo", "Jimbo.slack"));
+    controller.addCommitMockNotification(new Commit("abb23b4", "Merge of branch xyz", "rick", "rick.slack"));
+    controller.addCommitMockNotification(new Commit("adijefp", "branch xyz", "rick", "rick.slack"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "later commits", "frankjh3", "frankie.holzapfel"));
+
+    assertNotNull(sendMock(true));
+  }
+
+  @Test
+  public void oneMoreCommitsTest() throws IOException {
+    controller.addCommitMockNotification(new Commit("jdajfla", "first commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "second commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "later commits", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("dsalfhh", "other", "adfaafa", "fake.slack"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "first commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "second commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "later commits", "frankjh3", "frankie.holzapfel"));
+
+    assertNotNull(sendMock(true));
+  }
+
+  @Test
+  public void nMoreCommitsTest() throws IOException {
+    controller.addCommitMockNotification(new Commit("jdajfla", "first commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "second commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("dsalfhh", "other", "adfaafa", "fake.slack"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "later commits", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("dsalfhh", "other", "adfaafa", "fake.slack"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "first commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "second commit", "frankjh3", "frankie.holzapfel"));
+    controller.addCommitMockNotification(new Commit("dsalfhh", "other", "adfaafa", "fake.slack"));
+    controller.addCommitMockNotification(new Commit("jdajfla", "later commits", "frankjh3", "frankie.holzapfel"));
+
+    assertNotNull(sendMock(true));
+  }
```

Comment:
Is this series of tests actually sending messages to slack or to a mock slack? Meaning, do these actually show up in our slack?